dirsrvtests/tests/suites/clu/dsidm_role_test.py

```diff
@@ -289,11 +290,13 @@ def test_dsidm_role_list(topology_st, create_test_managed_role):
         5. Success
         6. Success
         7. Success
+        8. Success
     """
 
     standalone = topology_st.standalone
     args = FakeArgs()
     args.json = False
+    args.full_dn = False
```

Comment:
I think we need to update other tests' `FakeArgs` too by adding the `args.full_dn = False`

Reply:
> I think we need to update other tests' `FakeArgs` too by adding the `args.full_dn = False`

Yeah sounds good, and same for the JSON option....